sciter/event.py

```diff
@@ -380,12 +380,12 @@ def _element_proc(self, tag, he, evt, params):
 
     def script_exception_handler(self, func_name, exception):
         """
-        By default, just return passed in exception.
+        By default, just prints exception traceback to stderr and then returns it.
         Can be overridden to change script exception handling.
         """
         print("Python exception in `%s`: %s" % (func_name, repr(exception)))
```

Comment:
Is this redundant now? The traceback output contains the same info

Reply:
Looks like it is, yes